src/domain/accounts/primitives.ts

```diff
@@ -9,3 +9,11 @@ export const AccountStatus = {
   Active: "active",
   Locked: "locked",
 } as const
+
+export const AccountLimitsRange = {
```

Comment:
Yeah like this... everywhere you are hard coding strings that convey type information should be wrapped in something like this...

Oh and can we delete the comments? I think we can figure out how to add another value here without them.

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/1953/commits/a3583194056d0ac9151b0e33e74c8e397e31e472